<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Plugins::OpenSpeedShop::RemoteFileDialog</class>
 <widget class="QDialog" name="Plugins::OpenSpeedShop::RemoteFileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Open File</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,128,1">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="layPath">
     <item>
      <widget class="QLabel" name="lblPath">
       <property name="text">
        <string>Path</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtPath"/>
     </item>
     <item>
      <widget class="QToolButton" name="btnPathUp">
       <property name="text">
        <string>Go up</string>
       </property>
       <property name="icon">
        <iconset resource="../Resources/OpenSpeedShop.qrc">
         <normaloff>:/OpenSpeedShop/pathUp.svg</normaloff>:/OpenSpeedShop/pathUp.svg</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTreeWidget" name="treeWidget">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="headerHidden">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="lblFileName">
       <property name="text">
        <string>File name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtFileName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblFilter">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Filter</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cmbFilter">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="maximumSize">
      <size>
       <width>75</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Open</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Resources/OpenSpeedShop.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Plugins::OpenSpeedShop::RemoteFileDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Plugins::OpenSpeedShop::RemoteFileDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
